README.md

```diff
@@ -36,6 +36,15 @@ fxos.validateManifest(manifestUrl).then(function(result) {
 
 `manifestUrl` is a hosted manifest.webapp URL. (i.e. http://brittanystoroz.github.io/its-five-o-clock-somewhere/manifest.webapp). The promise resolves with a validated manifest ID, that can then be used to publish your application to the marketplace.
 
+## Validating a packaged application
+```javascript
+fxos.validatePackage(packagePath).then(function(result) {
+  console.log('Result: ', result);
```

Comment:
The manifest validation example does the same, which I more or less just copied...